### PR TITLE
Partial update of `VaDatePicker` visual style

### PR DESCRIPTION
**Pull request by RVitaly1978**
## Description
These changes are a partial solution to issue #1342.
I do not close this issue, as the requirements for it will expand (new features will be added).

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

File: packages/ui/src/components/va-date-picker/components/VaDatePickerCell.vue [hunk 1]
```diff
@@ -92,101 +89,79 @@ export default defineComponent({
...
+    cursor: not-allowed;
+    opacity: var(--va-date-picker-cell-opacity);
+
+    &::after { // Crossline
+      opacity: var(--va-date-picker-cell-opacity);
       border: none;
       height: 2px;
       width: 50%;
-      background: var(--va-date-picker-secondary);
+      background: currentColor;
```
Comment (LighthouseKeeperYN) [inline packages/ui/src/components/va-date-picker/components/VaDatePickerCell.vue:160]: What is currentColor?

File: packages/ui/src/components/va-date-picker/components/VaDatePickerCell.vue [hunk 2]
```diff
@@ -92,101 +89,79 @@ export default defineComponent({
...
-
-    &::before {
-      border: 2px solid var(--va-date-picker-focused-border-color);
-      opacity: $focus-border-opacity;
-    }
+    &:not(
+    .va-date-picker-cell_selected):not(
+      .va-date-picker-cell_readonly):not(
+        .va-date-picker-cell_disabled):not(
+          .va-date-picker-cell_other-month) {
```
Comment (LighthouseKeeperYN) [inline packages/ui/src/components/va-date-picker/components/VaDatePickerCell.vue:137]:
```suggestion
    &:not(.va-date-picker-cell_selected)
    :not(.va-date-picker-cell_readonly)
    :not(.va-date-picker-cell_disabled)
    :not(.va-date-picker-cell_other-month) {
```

**Review by LighthouseKeeperYN — APPROVED** [review]

File: packages/ui/src/components/va-date-picker/components/VaDatePickerCell.vue (hunk 2 above)
Comment (RVitaly1978) [reply packages/ui/src/components/va-date-picker/components/VaDatePickerCell.vue:133]: I checked this syntax but it doesn't work

File: packages/ui/src/components/va-date-picker/components/VaDatePickerCell.vue (hunk 1 above)
Comment (RVitaly1978) [reply packages/ui/src/components/va-date-picker/components/VaDatePickerCell.vue:156]: The `currentcolor` keyword refers to the value of the color property of an element

**Pull request merged**